magma/syntax/combinational.py

```diff
@@ -15,7 +15,7 @@
 from magma.config import get_debug_mode
 import itertools
 import typing
-from ast_tools.stack import _SKIP_FRAME_DEBUG_STMT
+from ast_tools.stack import _SKIP_FRAME_DEBUG_STMT, SymbolTable
```

Comment:
nit: if something is exported from a module (i.e. being imported by someone else) then it shouldn't have a leading `_` :)

Reply:
I am indifferent, but this is not my code.